### DISCUSSION TITLE
.exr image is clamped when -negate

### DISCUSSION THREAD

**Discussion by kelliray90** [ImageMagick/ImageMagick#6464] [Help] [answered]
### ImageMagick version

7.1.0-58 Q16-HDR

### Operating system, version and so on

windows 10

### Description

Hello, I used the command below to delete specific areas according mask. But in the result, the HDR data of the sun is clamped:

```
m mountain.exr ( mask.png -negate ) -alpha off -compose copy_opacity -composite result.exr
```
Download image here: [Image.zip](https://github.com/ImageMagick/ImageMagick/files/11956335/Image.zip)

(The below images was captured in Photoshop. I adjust the exposure to check the HDR data)
The orig image:
![Screenshot 2023-07-05 164250](https://github.com/ImageMagick/ImageMagick/assets/28614684/e2fd05b2-aa27-4145-901e-5ecfd4ffe462)
After run the above command and decrease the exposure to check HDR data:
![Screenshot 2023-07-05 164237](https://github.com/ImageMagick/ImageMagick/assets/28614684/4212f5eb-0220-4ca7-9f3f-ad9834b55372)

**Comment by snibgo** [comment, marked as answer]
By default, `-composite` clamps. To prevent this, use "`-define compose:clamp=off`" before the `-composite`.

**Reply by kelliray90** [reply]
It worked! Thank you so much!!
